tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -643,3 +643,90 @@ def forward_xp(self, inputs, xp):
 def test_split_invalid(xp, shape, indices_or_sections, axis):
     a = array_utils.create_dummy_ndarray(xp, shape, 'float32')
     return xp.split(a, indices_or_sections, axis)
+
+
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    chainer.testing.product({'shapes': [
+        (1,),
+        (1, 1),
+        (1, 1, 1),
+        (2, 2, 2, 2),
+    ],
+        'dtype': chainerx.testing.dtypes.float_dtypes
```

Comment:
Please use `all_dtypes` instead of `float_dtypes`.

Reply:
Oh sorry, was using that while testing, will change it to all. Thanks